ADOPTERS.md

```diff
@@ -0,0 +1,3 @@
+# Adopters
+
+This is a list of adopters of using PowerShell in production or in their products (in alphabetical order):
```

Comment:
We should have some structure about what information we want from the adopters.

Reply:
I would suggest at minimum, company name, contact name...

Reply:
Perhaps add the skeleton of a Markdown table?

Reply:
Looking at other examples, they don't have that in this doc.  I'm going to reach out to some internal partners first with an example and that will set the structure.

Reply:
I guess I can add instructions/template as a comment in the markdown so it isn't rendered.